sda/status.go

```diff
@@ -0,0 +1,36 @@
+package sda
+
+//Status is the status server
+type Status map[string]string
+
+//StatusReporter is the interface that all things that want to return a status will implement
+type StatusReporter interface {
+	GetStatus() Status
+}
+
+//statusReporter struct is a struct
```

Comment:
don't comment what is obvious, but write what can help your fellow programmers understand what is _special_ and is needed to know to understand the structure. Here it could be:

`// statusReporter holds a map of all StatusReporters`

And don't forget the 'space' between the `//` and your text!